src/components/activities/editPatientActivity/EditPatientActivity.tsx

```diff
@@ -128,34 +134,42 @@ const EditPatientActivity: FunctionComponent<TProps> = ({
                   patient.data?.secondName
                 }`}
               </div>
-              <PatientDataForm
-                fields={updateFields(initialFields, patient?.data)}
-                profilePicture={patient.data?.blobPhoto}
-                onSubmit={onSubmit}
-                submitButtonLabel={t("common.submit")}
-                resetButtonLabel={t("common.reset")}
-                isLoading={isLoading}
-                shouldResetForm={shouldResetForm}
-                resetFormCallback={resetFormCallback}
-              />
+              <PermissionWrapper permissions={["patient.update"]}>
+                <PatientDataForm
+                  fields={updateFields(initialFields, patient?.data)}
+                  profilePicture={patient.data?.blobPhoto}
+                  onSubmit={onSubmit}
+                  submitButtonLabel={t("common.submit")}
+                  resetButtonLabel={t("common.reset")}
+                  isLoading={isLoading}
+                  shouldResetForm={shouldResetForm}
+                  resetFormCallback={resetFormCallback}
+                />
+              </PermissionWrapper>
```

Comment:
I think that all the jsx should be wrapped inside the permission check component. It's not wrong leave as is but I prefer that all the "protected" component (even If I cannot access to them directly as for example a modal) be stay under the wrapper. 

Please follow this here and in the other files.